src/game_renderer/Shape.cpp

```diff
@@ -10,51 +10,49 @@
 using namespace std;
```

Comment:
many of them are functional you may need to look at the changes that have been made here